azurelinuxagent/common/telemetryevent.py

```diff
@@ -20,6 +20,57 @@
 from azurelinuxagent.common.datacontract import DataContract, DataContractList
 from azurelinuxagent.common.version import AGENT_NAME
 
+class CommonTelemetryEventSchema(object):
+
+    # Common schema keys for both tables
```

Comment:
This is the common schemas for all the tables that we have in Kusto (GAExtensionEvents, GAGenericLogs, GAPerfMetrics). Do you want me to add a comment for these 3 tables? I'm not sure if that's required mainly because we might add/remove tables later on so the comment would get outdated. We can discuss this offline if you want.

Reply:
nvm added table names to the comment.